query-engine/connector-test-kit-rs/query-engine-tests/tests/new/metrics.rs

```diff
@@ -20,8 +20,8 @@ mod metrics {
 
         let json = runner.get_metrics().to_json(Default::default());
         // We cannot assert the full response it will be slightly different per database
-        let total_queries = get_counter(&json, "query_total_queries");
-        let total_operations = get_counter(&json, "query_total_operations");
+        let total_queries = get_counter(&json, "prisma_datasource_queries_total");
+        let total_operations = get_counter(&json, "prisma_client_queries_total");
```

Comment:
Nit: I think optimally we should clean up the local variable names as well as that will make it a _lot_ maintainable going forward.

Reply:
Ah, it's tests only. 
Still.

Reply:
I agree and originally tried to do that. But it's really tricky. The way metrics are scattered across so many crates means it creates some weird dependencies and I couldn't get it to work nicely.